app/templates/frameworks/dashboard.html

```diff
@@ -159,9 +159,15 @@
             <ul>
               <li>
                 <a href="{{ url_for('.download_supplier_file', filepath='g-cloud-7-supplier-pack.zip') }}"><span>Download guidance and legal documentation (.zip)</span></a>
+                {% if last_modified.supplier_pack %}
+                  <p class="hint">Last updated <time datetime="{{ last_modified.supplier_pack }}">{{ last_modified.supplier_pack|dateformat }}</time></p>
+                {% endif %}
               </li>
               <li>
                 <a href="{{ url_for('.framework_updates') }}"><span>Read updates and ask clarification questions</span></a>
+                {% if last_modified.supplier_updates %}
+                  <p class="hint">Last updated <time datetime="{{ last_modified.supplier_updates }}">{{ last_modified.supplier_updates|dateformat }}</time></p>
```

Comment:
These should just be a `div`—doesn't make sense to have paragraphs inside list items.